test/integration/test_integration_xarray_plugin.py

```diff
@@ -0,0 +1,29 @@
+import os.path
+
+import pytest
+
+from test.conftest import TEST_INPUT_DATA_DIR
+
+xr = pytest.importorskip("xarray", minversion="0.17.1.dev0")
+
+
+def test_xarray_open_dataset():
```

Comment:
Please use the `tmp_path` fixture [ref](https://docs.pytest.org/en/stable/tmpdir.html) to write the test output netcdf file.

Reply:
w00t! I didn't intend to leave the saving of the files in the code. I'll remove the calls `to_netcdf`.